server/server.js

```diff
@@ -1,40 +1,52 @@
+// importing needed modules
 import express from "express";
 import cors from "cors";
 import "dotenv/config";
+import https from "https";
+import path from "path";
+import { fileURLToPath } from "url";
+import fs from "fs";
+// importing custom sequelize class
 import databaseHelper from "./databasehelper.js";
 
+// create express app
 const APP = express();
+APP.use(express.json());
+APP.use(cors());
 
-// dev environment vars
-// const DATABASE = process.env.DATABASE;
-// const USERNAME = process.env.USERNAME;
-// const PASSWORD = process.env.PASSWORD;
-// const PORT = 5000;
-
-// prod environment vars
-// const DATABASE = process.env.PGDATABASE;
-// const USERNAME = process.env.USER;
-// const PASSWORD = process.env.PGPASSWORD;
-const PORT = process.env.PGPORT;
-
-// const dbhelper = new databaseHelper(DATABASE, USERNAME, PASSWORD);
+// get directory name
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
-const dbhelper = new databaseHelper();
+// current app environment
+const APP_ENV = process.env.APP_ENV;
```

Comment:
This will all be simpler if you just standardize on DATABASE_URL or DB_URL (whatever your host gives you), and use it locally too.

Reply:
Fixed!